test/features/email/presentation/controller/single_email_controller_test.dart

```diff
@@ -267,16 +269,19 @@ void main() {
     final blobId = Id('abc123');
     final calendarEvent = CalendarEvent();
 
+    setUp(() {
+      when(mailboxDashboardController.selectedEmail).thenReturn(Rxn(null));
+      when(mailboxDashboardController.emailUIAction).thenReturn(Rxn(null));
+      when(mailboxDashboardController.viewState).thenReturn(Rx(Right(UIState.idle)));
+      singleEmailController.onInit();
```

Comment:
why not use it in `test` method?

Reply:
common setup for group of tests should be inside the group's `setUp` method. Also to avoid code duplication

Reply:
We should not use it like that.

Reply:
Done